tardis/model/tests/data/model_full_old_config.yml

```diff
@@ -33,7 +33,7 @@ model:
 
   abundances:
     type: file
-    filename: abund.dat
+    filename: custom_abundance.dat
```

Comment:
Add _tardis to the name for custom abundance.dat and density.dat since we get custom abundance/density from other sources as well with different conventions.